io/disk/scsi_add_remove.py.data/README.txt

```diff
@@ -1,7 +1,7 @@
-This Test removes and adds back a scsi device in all the specified PCI domains
+This Test removes and adds back a scsi device in all the specified wwids
 specified in the 'multiplexer' file.
 This test needs to be run as root.
 
 Inputs Needed (in 'multiplexer' file):
 --------------------------------------
-PCI_devices -   PCI Device entry got from 'lspci' command.
+wwids -   wwids can be fetch from 'multipath -ll' command.
```

Comment:
or lsscsi command.
There is an option to get that. please document that as well.